src/common/src/array/utf8_array.rs

```diff
@@ -26,7 +26,7 @@ use crate::array::ArrayBuilderImpl;
 use crate::buffer::{Bitmap, BitmapBuilder};
 
 /// `Utf8Array` is a collection of Rust Utf8 `String`s.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
I'm not sure whether it's okay to derive `Clone` for all kinds of arrays. 🤔 cc @skyzh